vllm/platforms/__init__.py

```diff
@@ -32,13 +32,8 @@
 is_rocm = False
 
 try:
-    import amdsmi
-    amdsmi.amdsmi_init()
-    try:
-        if len(amdsmi.amdsmi_get_processor_handles()) > 0:
-            is_rocm = True
-    finally:
-        amdsmi.amdsmi_shut_down()
+    import torch
+    is_rocm = torch.version.hip is not None
```

Comment:
I think this may not be as robust as the original code. It is possible to have PyTorch installed for ROCm but no available GPUs.

Reply:
Also see the comment at the top of the file.

Reply:
Thanks for your review，I will roll back code here. Is it a better way for adding func `is_hip()` and  `elif is_hip: from .rocm import RocmPlatfom`?

Reply:
I am not familiar with AMD, so i don`t know if  there is a situation where `is_rocm` is true, but `is_hip` is false? or i just need to roll back this code.

Reply:
I found `HIP is automatically installed during the ROCm installation` at https://rocm.docs.amd.com/projects/HIP/en/latest/install/install.html, so I roll back this code.